lib/system/excpt.nim

```diff
@@ -320,6 +337,11 @@ proc raiseExceptionAux(e: ref Exception) =
       quitOrDebug()
     else:
       pushCurrentException(e)
+      discard popCurrentExceptionEx # small hack to trigger dependency for codegen
```

Comment:
There must be a clean(er) way to do this.

Reply:
The problem is that I can't emit ``popCurrentExceptionEx`` dependency during ``getTypeDesc`` call for ``Exception`` type as ``Exception`` type is used inside ``popCurrentExceptionEx``. Endless loop.

The only idea I have is to introduce something like ``TCGen.typeStack``, but for procs/syms so I can push ``popCurrentExceptionEx`` into it and the proc gets emitted in ``writeModule`` as finalization step

More ideas welcome